pixano/app/api/items.py

```diff
@@ -474,8 +472,7 @@ def find_top_parent_entity(entity: Entity) -> Entity | None:
                 kept_track_ids.append(track_id)
```

Comment:
kept_track_ids seems useless right now, but it may become usefull later, so let's keep it

Reply:
yes everything will be refactor at some moint anyway